sunpy/net/dataretriever/tests/test_noaa.py

```diff
@@ -60,13 +62,11 @@ def test_fetch(time, instrument):
     assert len(download_list) == len(qr1)
 
 
-@pytest.mark.remote_data
-@pytest.mark.parametrize(
-    "time, instrument",
-    [(a.Time("2012/10/4", "2012/10/6"), a.Instrument('noaa-indices')),
-     (a.Time('2013/10/5', '2013/10/7'), a.Instrument('noaa-indices'))])
-def test_fido(time, instrument):
-    qr = Fido.search(time, instrument)
+@mock.patch('sunpy.net.dataretriever.sources.noaa.NOAAIndicesClient._get_default_uri',
+            return_value='https://example.com')
```

Comment:
Any change the return value could the expected one?

Reply:
The original test returns a real result from the search.

We need this result since we need to create a second test that checks that the mocked response is still correct in the future.